frontend/src/metabase/components/BulkActionBar.styled.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "@emotion/styled";
```

Comment:
Since `BulkActionBar` is no longer being used in `BulkActions`, I moved the styles from `BulkActions.styled.tsx` to this new file colocated with the actual component.